### PR TITLE
Hw 21.1 jdbc(item dao jdbc impl)

### PR DESCRIPTION
**Pull request by Valdes2**

### CODE REVIEW

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+import mate.academy.internetshop.dao.ItemDao;
+import mate.academy.internetshop.db.Storage;
+import mate.academy.internetshop.lib.Dao;
+import mate.academy.internetshop.model.Item;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ItemDaoJdbcImpl extends AbstractDao<Item> implements ItemDao {
+    private static Logger logger = Logger.getLogger(ItemDaoJdbcImpl.class);
+    private static String DB_NAME = "internetshop";
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
    private static final String DB_NAME = "internetshop";
```

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        String query = String.format("INSERT INTO %s.items (name, price) values(?, ?)", DB_NAME);
+
+        try {
+            preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, item.getName());
+            preparedStatement.setDouble(2, item.getPrice());
+            preparedStatement.executeUpdate();
+            return item;
+        } catch (SQLException e) {
+            logger.warn("Can`t create item", e);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
            logger.error("Can`t create item", e);
```

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        } finally {
+            if (preparedStatement != null) {
+                try {
+                    preparedStatement.close();
+                } catch (SQLException e) {
+                    logger.warn("Can`t close statement", e);
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]:
```suggestion
                    logger.error("Can`t close statement", e);
```

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+                }
+            }
+        }
+
```
Comment (Sofasmile) [inline src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: remove empty line

**Review by Sofasmile — APPROVED** [review]
make small fixes

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 4 above)
Comment (Valdes2) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: fixed

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 3 above)
Comment (Valdes2) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: fixed

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 2 above)
Comment (Valdes2) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: fixed

File: src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java (hunk 1 above)
Comment (Valdes2) [reply src/main/java/mate/academy/internetshop/dao/jdbc/ItemDaoJdbcImpl.java]: fixed

**Pull request merged**